index.html

```diff
@@ -392,21 +392,39 @@ <h2>Terminology</h2>
       <section id="stakeholders">
         <h2>Stakeholders and Roles</h2>
         <p>
-          The following stakeholders and actors were identified
-          when the use cases have been collected and requirements were identified.
-          Note that these stakeholders and roles may overlap in some use cases.
+	  Whenever possible, stakeholders should be identified using the terms
+	  defined in the WoT Security and Privacy Guidelines [[wot-security]].
+	  For convenience these terms are listed here, but please refer to that
+	  document for full definitions:
+	</p>
+	<ul>
+          <li id="STAKEHOLDER-Device-Manufacturer">Device Manufacturer</li>
+          <li id="STAKEHOLDER-System-Provider">System Provider</li>
+          <li id="STAKEHOLDER-System-Integrator">System Integrator</li>
+          <li id="STAKEHOLDER-System-Installer">System Installer</li>
+          <li id="STAKEHOLDER-System-User">System User</li>
+          <li id="STAKEHOLDER-System-Owner">System Owner</li>
+          <li id="STAKEHOLDER-System-Maintainer">System Maintainer</li>
+        </ul>
+        <p>
+          The following additional stakeholders and roles were identified
+
+          when the use cases were collected, or are defined in other WoT documents:
         </p>
+        <ul>
+          <li id="STAKEHOLDER-Device-Owner">Device Owner: A special case of System Owner when not all devices in a system may have the same owner, or when the focus is on the ownership of a specific device.</li>
+          <li id="STAKEHOLDER-Device-User">Device User: The human interacting with the actual physical user interface of a device, or whose environment may be sensed or modified by the device.  May also be used when referring to the data or network service provided by a particular device.</li>
+          <li id="STAKEHOLDER-Cloud-Provider">Cloud Provider: An organization providing remote computing services available over the network.</li>

```

Comment:
> When I wrote "the network" I really meant "the internet", not some random LAN or corporate network. Suggest changing this to "the internet" instead of "a network".

I think "the internet" is actually too restrictive. Why should there be only one "cloud"? Why shouldn't it be "some random LAN or corporate network"?
```suggestion
          <li id="STAKEHOLDER-Cloud-Provider">Cloud Provider: An organization providing remote computing services available over a network.</li>
```